index.html

```diff
@@ -7,100 +7,114 @@
+            <p>
+              <span class="line line-2">But for some reason, you and I react the exact same way to water.</span>
+              <span class="line line-2">We swallow it too fast, we choke. We get some in pur lungs, we</span>
+              <span class="line line-2">drown. However unreal it may seem, we are connected, you and I.</span>
+              <span class="line line-2">We're on the same curve, just on opposite ends.</span>
+            </p>
         </div>
+        <ul class="album">
+          <ul><div class="arrow"><img src="./assets/arrow-left.png" alt="arrow"></div></ul>
+          <ul><img class="image" src="./assets/tile-5.png" alt="arrow"></ul>
+          <ul><img class="image" src="./assets/tile-1.png" alt="arrow"></ul>
+          <ul><img class="image" src="./assets/tile-2.png" alt="arrow"></ul>
+          <ul><img class="image" src="./assets/tile-3.png" alt="arrow"></ul>
+          <ul><img class="image" src="./assets/tile-4.png" alt="arrow"></ul>
+          <ul><div class="arrow"><img src="./assets/arrow-right.png" alt="arrow"></div></ul>
+        </ul>
       </div>
+    </div>
   </section>
-  <section class="sectionII">
-    <div class="first_box_middle">
-      <p class="estilo_titulo">That's Cesar's world <span>HARSH REALITY</span></p>
-        <p class="line">
-        Like you, I used to think the word was this<br> 
-        great place where everybody lived by the same<br>
-        standards I did, then some kid with a nail<br>
-        showed me I was living in his world, a world<br>
-        where chaos rules not order, a world where<br>
-        righteousness is not rewarded. That's Cesar's<br>
-        world, and if you're not willing to play by his<br>
-        rules, then you're gonna have to pay the price.<br>
-      </p>
+  <section class="section sectionII">
+    <div class="container third-container">
+      <div class="first_box_middle">
+        <p class="header-table header-3">That's Cesar's world <span>HARSH REALITY</span></p>
+        <p class="line.1">
+        <span class="line">Like you, I used to think the word was this</span>
+        <span class="line">great place where everybody lived by the same</span>
+        <span class="line">standards I did, then some kid with a nail</span>
+        <span class="line">showed me I was living in his world, a world</span>
+        <span class="line">where chaos rules not order, a world where</span>
+        <span class="line">righteousness is not rewarded. That's Cesar's</span>
+        <span class="line">world, and if you're not willing to play by his</span>
+        <span class="line">rules, then you're gonna have to pay the price.</span>
+        </p>
+      </div>
+      <div class="second_box_middle">
+        <table border = 1>
+        <caption colspan="4" class="header-table">List style: Circles. <span>FEATURING FEATURES</span></caption>
+          <tr>
+            <td class="one">
+              <P>USER <br><span>PROFILES</span></P>
```

Comment:
```suggestion
              <p>USER <span>PROFILES</span></p>
```

No uses `<br>` y la etiqueta html va en minuscula.